alf/trainers/policy_trainer.py

```diff
@@ -317,6 +317,14 @@ def train(self):
                 "Use `kill -%s %s` to request checkpoint during training." %
                 (int(signal.SIGUSR2), self._pid))
 
+        self._video_clip_requested = False
+        if threading.current_thread() == threading.main_thread():
+            signal.signal(signal.SIGRTMIN, self._request_video_clip)
+            # kill -34 PID
+            logging.info(
+                "Use `kill -%s %s` to request video-clip during training." %
```

Comment:
Also include information about where is the video file located.

Reply:
Thanks for the comment. Added. Now will output info as follows:

<img width="1169" alt="image" src="https://github.com/user-attachments/assets/be23fb26-ef66-45e3-ac80-01ec67520fee" />